src/pages/LoginForm.tsx

```diff
@@ -1,104 +1,22 @@
-import { useContext, useState, useEffect } from 'react';
-import { useNavigate } from 'react-router';
-import { useApolloClient } from '@apollo/client';
-import { Box, Button, Chip, Stack } from '@mui/material';
-
-import AccountTreeIcon        from '@mui/icons-material/AccountTree';
-import FormatListBulletedIcon from '@mui/icons-material/FormatListBulleted';
-import ViewStreamIcon         from '@mui/icons-material/ViewStream';
-import CampaignIcon           from '@mui/icons-material/Campaign';
-import EditIcon from '@mui/icons-material/Edit';
-
-import { UserContext, UserContextProps, UserProps } from "../contexts/UserContext";
-import  AnnouncementBox  from '../components/AnnouncementBox';
-
-function MenuButton({onClick, navigateTo, children}) {
-    const navigate = useNavigate();
-    return (
-        <Button variant="contained" onClick={onClick ? onClick : () => navigate(navigateTo)}  sx={{ m: 2, width: '210px', textTransform: 'none' }}>
-            <Box sx={{ width: '100%', display: 'flex', justifyContent: 'space-between', alignItems: 'center'}}>
-                {children[0]}
-                <Box sx={{width: '100%'}}>{children.slice(1)}</Box>
-            </Box>
-        </Button>
-    );
-}
+import { useContext } from 'react';
+import { Navigate } from 'react-router';
+import { Box, Button } from '@mui/material';
+import { UserContext, UserContextProps } from "../contexts/UserContext";
 
 export default function LoginForm() {
-  const apolloClient = useApolloClient();
-  const navigate = useNavigate();
   const userContext: UserContextProps = useContext(UserContext);
-  const userProps : UserProps = userContext.userProps;
+  const userProps = userContext.userProps;
 
-  function logout() {
-      // Technically we should clear the Apollo client cache on logout, but right now, even without the resetStore()
```

Comment:
Ah sorry, missed this the first time around - could you add this comment back in to the new/relocated function? :pray: